dist/src/main/java/io/camunda/zeebe/broker/health/BrokerLiveHealthIndicator.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.1. You may not use this file
+ * except in compliance with the Zeebe Community License 1.1.
+ */
+package io.camunda.zeebe.broker.health;
+
+import io.camunda.zeebe.broker.SpringBrokerBridge;
+import io.camunda.zeebe.broker.system.monitoring.BrokerHealthCheckService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.actuate.availability.LivenessStateHealthIndicator;
+import org.springframework.boot.actuate.health.Health;
+import org.springframework.boot.availability.ApplicationAvailability;
+import org.springframework.boot.availability.AvailabilityState;
+import org.springframework.boot.availability.LivenessState;
+import org.springframework.stereotype.Component;
+
+@Component
+public final class BrokerLiveHealthIndicator extends LivenessStateHealthIndicator {
```

Comment:
I assume this is a :thought_balloon: because this is the current state, right? I think the main question here is what's unavailable for us. If a partition is unavailable, but others are, do we consider the system available (w.r.t. to our SLA)? If so, then restarting/removing the node is not a good idea.

Reply:
It is more of a :x: , but since I don't have a clear idea what to do here I put it as :thought_balloon: 
Previously it was `/health` which do not say anything about liveness. But now is is `actuator/health/liveness` which would mean that we recommend people to use it as liveness probe in kubernetes. But ideally IMO, it shouldn't be used as liveness probe.

Reply:
Oh, I see! Hm, we can redirect it to `/health` here, which would include both `liveness` and `readiness` of course. Right now it would be OK, but in the future it might be problematic if liveness changes meaning.

BTW we do use `/health` as the liveness probe right now I think?

Reply:
Yes and No. It is set in helm, but is also disabled by default. In SaaS we don't use it. See our past discussion [here](https://camunda.slack.com/archives/C037RS2JHB8/p1693487963512249).

Reply:
Ah, fair. So merging would have no impact at the moment.

To keep the behavior the same, for now, I could create a sub group called `broker-health`? Basically we can create a health sub-group (like the startup) and have it redirect there (since `/actuator/health` returns the health for all of them).

And liveness would simply not be used for now? :thinking:

Reply:
`broker-health` is a poor name though. I'm not sure what to call it - `broker-state`? :thinking:

Reply:
> To keep the behavior the same, for now, I could create a sub group called `broker-health`? Basically we can create a health sub-group (like the startup) and have it redirect there (since `/actuator/health` returns the health for all of them).

If you can do that it would be great :+1:

> And liveness would simply not be used for now? 🤔

Yes. If not configured it always returns true, right? We could also think about what other metrics are useful for liveness. But not in the scope of this PR.

Reply:
> `broker-health` is a poor name though. I'm not sure what to call it - `broker-state`? 🤔

`broker-status`?